frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {useEffect} from 'react';
+import styled from "styled-components";
+import {Card} from '@material-ui/core';
+import {MatchItem} from "../../molecules/MatchItem/MatchItem";
+import {MIN_CARD_COMPONENT_WIDTH} from "../../atoms/constants/sizes";
+import {MatchListItem} from "./MatchListItem";
+import {MatchesListRestApi} from "../../../restapi/matches-list";
+import {MatchesListDto} from "./MatchesListDto";
+import {MatchStatus} from "./MatchStatus";
+
+const StyledMatchesList = styled(Card)({
+    width: MIN_CARD_COMPONENT_WIDTH,
```

Comment:
Zastanawia mnie to, ze wszedzie tutaj ustawiamy. Czy nie powinnismy po prostu ograniczyc jakiegos kontenera a karty by sie do niego dopasowywaly?

Reply:
Można, ale też czasem (np. w przypadku Tomka) musimy tą wartość użyć do elementu wewnętrznego. Dodatkowo elementy mniejsze mają niezależność dzięki tej wartości